salt/utils/minions.py

```diff
@@ -211,14 +211,15 @@ def _check_glob_minions(self, expr, greedy):  # pylint: disable=unused-argument
         '''
         return fnmatch.filter(self._pki_minions(), expr)
 
-    def _check_list_minions(self, expr, greedy):  # pylint: disable=unused-argument
+    def _check_list_minions(self, expr, greedy, ignore_missing=False):  # pylint: disable=unused-argument
         '''
         Return the minions found by looking via a list
         '''
         if isinstance(expr, six.string_types):
             expr = [m for m in expr.split(',') if m]
         minions = self._pki_minions()
-        return [x for x in expr if x in minions]
```

Comment:
Oh, I think I mixed something up here. Maybe `2017.7` is too early and we only need that for `2018.3`. Seems like the `missing` key was not yet introduced. I'm re-checking this. Sorry!